cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateStreamCommand.java

```diff
@@ -45,14 +57,28 @@ public CreateStreamCommand(StreamClient streamClient, CLIConfig cliConfig) {
   @Override
   public void perform(Arguments arguments, PrintStream output) throws Exception {
     String streamId = arguments.get(ArgumentName.NEW_STREAM.toString());
+    StreamProperties streamProperties = null;
 
-    streamClient.create(Id.Stream.from(cliConfig.getCurrentNamespace(), streamId));
+    if (arguments.hasArgument(ArgumentName.LOCAL_FILE_PATH.toString())) {
+      File file = new File(arguments.get(ArgumentName.LOCAL_FILE_PATH.toString()));
+      if (!file.isFile()) {
```

Comment:
It's better not to check in here, but letting the creation of Reader to check and fail with an `IOException` below.